app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -224,10 +224,15 @@ public void onValueChanged(@NonNull Boolean loading) {
                     progressView.setProgress(5);
                     progressView.setVisibility(View.VISIBLE);
                 } else {
-                    backgroundTransitionGroup.startTransition(ANIMATION_DURATION);
 
-                    progressView.setProgress(progressView.getMax());
-                    progressView.setVisibility(View.GONE);
+                    if(progressView.getVisibility() == View.VISIBLE) {
```

Comment:
\[Checkstyle\] ERROR: 'if' is not followed by whitespace\.

Reply:
@sputnikci it's fixed thank you